Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -186,6 +198,7 @@ - (SentryIntegrationOption)integrationOptions
 
 - (void)uninstall
 {
+    [SentrySDK.currentHub unregisterSessionListener:self];
```

Comment:
`m`: Maybe it's good to call `uninstall` also from `dealloc` so we ensure to clear this up when this class gets deallocated.